doxygen/lang/900_release_notes.dox.tmpl

```diff
@@ -360,6 +362,7 @@
       - @ref Qore::ReadOnlyFile::readBinaryFile()
       - @ref Qore::ReadOnlyFile::readTextFile()
     - fixed a bug in @ref Qore::GetOpt::parse(softlist), @ref Qore::GetOpt::parse2(softlist), and @ref Qore::GetOpt::parse3(softlist) where call-by-value and copy-on-write semantics were not enforced and a shared list argument could be modified
+    - fixed a bug in zoneinfo file parsing where invalid bands with no UTC offset changes against the previous band could cause invalid date/time values to be displayed by Qore for dates on the invalid transition
```

Comment:
well we are not going to make another release of 0.8.11 so nobody would see the release notes - the updates are only for qorus 3.0.4